science/Makefile

```diff
@@ -177,7 +177,11 @@
     SUBDIR += opensph
     SUBDIR += opsin
     SUBDIR += orthanc
+<<<<<<< HEAD
     SUBDIR += orthanc-webviewer
+=======
+    SUBDIR += orthanc-postgresql
+>>>>>>> 21b9f5c6fea6 (science/orthanc-postgresql: Revive port)
```

Comment:
unresolved conflict.

Reply:
Sorry - this port should still be wip, and I accidentally tagged you.